quetz/config.py

```diff
@@ -117,6 +117,17 @@ class Config:
             ],
             required=False,
         ),
+        ConfigSection(
+            "azure_blob",
+            [
+                ConfigEntry("account_name", str, default=""),
+                ConfigEntry("account_access_key", str, default=""),
+                ConfigEntry("conn_str", str, default=""),
```

Comment:
is this an commonly used abbreviation?

Reply:
Yes, it's often in arguments of the official client libraries by Azure.